<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ApplyDialog</class>
 <widget class="QDialog" name="ApplyDialog">
  <property name="windowModality">
   <enum>Qt::WindowModality::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>698</width>
    <height>499</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ALT Packages - Transaction Approval</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="installVerticalLayout">
     <item>
      <widget class="QLabel" name="installLabel">
       <property name="text">
        <string>Packages to install (%1):</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QListWidget" name="installPackages"/>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <layout class="QVBoxLayout" name="removeVerticalLayout">
     <item>
      <widget class="QLabel" name="removeLabel">
       <property name="text">
        <string>Packages to remove (%1):</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QListWidget" name="removePackages"/>
     </item>
    </layout>
   </item>
   <item row="1" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="applyHorizontalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>503</width>
         <height>28</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>Cancel</string>
       </property>
       <property name="icon">
        <iconset theme="QIcon::ThemeIcon::ProcessStop"/>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="applyButton">
       <property name="text">
        <string>Apply</string>
       </property>
       <property name="icon">
        <iconset theme="emblem-default"/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
